cleanup.sh

```diff
@@ -3,25 +3,19 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: MIT-0
 
-#Remove test deployment
-cd devopsguru_eks_test
-sh uninstall_chart.sh
-cd ..
+export AWS_DEFAULT_REGION=us-east-1
+export AWS_PAGER=""
 
-sleep 300
+echo "Remove test deployment"
+(cd devopsguru_eks_test && sh uninstall_chart.sh)
 
-#Cleanup ECR repo
-cd ecr
-sh remove_repo.sh
-cd ..
+sleep 300
```

Comment:
Could we add a `echo "Sleep for 5 min..."` so that user knows what is going on

Reply:
Good point, I will add that.